### DISCUSSION TITLE
How tall are you?

### DISCUSSION THREAD

**Discussion by tallguyjenks** [BryanJenksCommunity/FAQ#1] [Q&A] [answered]
I'm 6'6" or 198cm for you metric folks ._.

**Comment by tallguyjenks** [comment, marked as answer]
Converted from issue

**Comment by nickwarters** [comment]
> I'm 6'6" or 198cm for you metric folks

Jeez! I might start branding myself as shortmanwarters - I'm 5'2"

**Reply by tallguyjenks** [reply]
LOL excellent handle 🤣
